src/Notifier.js

```diff
@@ -83,15 +83,26 @@ var Notifier = {
             });
             global.focus();
         };
-        
-        /*var audioClip;
-        
-        if (audioNotification) {
-            audioClip = playAudio(audioNotification);
-        }*/
+
+        var playAudio = function() {
+            var e = document.getElementById("messageAudio");
+            if (e) {
+                e.load();
+                e.play();
+                return e;
+            }
+        };
+
+        var audioClip;
+        if (actions.tweaks.sound) {
+            audioClip = playAudio();
+        }
 
         global.setTimeout(function() {
             notification.close();
+            if (audioClip) {
+                audioClip.pause();
```

Comment:
Actually this probably isn't necessary, and would cause the notification sound to stop mid-way if a second notif came in and then a previous one hit the timeout.